test/runtests.jl

```diff
@@ -6,7 +6,7 @@ fatalerrors = length(ARGS) > 0 && ARGS[1] == "-f"
 quiet = length(ARGS) > 0 && ARGS[1] == "-q"
 anyerrors = false
 
-using Test, Random, DataFrames
+using DataFrames, Dates, Test, Random
```

Comment:
Isn't this redundant with the next change?

Reply:
This is needed since `show.jl` runs the test inside a submodule `TestShow`. Without this the Date tests will fail with:
```
Test showing special types: Test Failed at /Users/omus/.julia/dev/DataFrames/test/show.jl:320
  Expression: sprint(show, df) == "1×3 DataFrame\n│ Row │ a          │ b                   │ c     │\n│     │ Date       │ DateTime            │ Day   │\n├─────┼────────────┼─────────────────────┼───────┤\n│ 1   │ 2020-02-11 │ 2020-02-11T15:00:00 │ 1 day │"
   Evaluated: "1×3 DataFrame\n│ Row │ a          │ b                   │ c      │\n│     │ Dates.Date │ Dates.DateTime      │ Dates… │\n├─────┼────────────┼─────────────────────┼────────┤\n│ 1   │ 2020-02-11 │ 2020-02-11T15:00:00 │ 1 day  │" == "1×3 DataFrame\n│ Row │ a          │ b                   │ c     │\n│     │ Date       │ DateTime            │ Day   │\n├─────┼────────────┼─────────────────────┼───────┤\n│ 1   │ 2020-02-11 │ 2020-02-11T15:00:00 │ 1 day │"
Stacktrace:
 [1] top-level scope at /Users/omus/.julia/dev/DataFrames/test/show.jl:320
 [2] top-level scope at /Users/omus/Development/Julia/1.3/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
 [3] top-level scope at /Users/omus/.julia/dev/DataFrames/test/show.jl:284
```